Tests/SnapKitTests/Tests.swift

```diff
@@ -494,8 +494,8 @@ class SnapKitTests: XCTestCase {
     
     #if os(iOS) || os(tvOS)
     func testEdgesToMargins() {
-        var fromAttributes = Set<NSLayoutAttribute>()
-        var toAttributes = Set<NSLayoutAttribute>()
+        var fromAttributes = Set<NSLayoutConstraint.Attribute>()
```

Comment:
I think you can just use `LayoutAttribute` instead: https://github.com/SnapKit/SnapKit/blob/15beb5221834e220f099a87beb80cc9d9abb16cf/Source/Typealiases.swift#L33

Otherwise you'd break support for Swift 4 IMO